ts/private/ts_project_worker.js

```diff
@@ -744,4 +744,4 @@ if (require.main === module && worker.runAsWorker(process.argv)) {
     }
 }
 
-module.exports = {createFilesystemTree: createFilesystemTree, emit: emit};
+module.exports.__do_not_use_test_only__ = {createFilesystemTree: createFilesystemTree, emit: emit, workers: workers};
```

Comment:
Does anything other then tests import this file? Is this necessary?

Reply:
this is not part of the public API. so no one should depend on it except test in this repo.

Reply:
Shouldn't no one even depend on the file though? I don't really see the value in adding a funny name 🤷